CHANGELOG.md

```diff
@@ -1,6 +1,47 @@
 Change log
 ==========
 
+1.5.0 (2015-10-13)
+------------------
+
+Major Features
+
+-   Compose is now available on windows.
+-   Environment variable can be used in the compose file. See
+    https://github.com/docker/compose/blob/master/docs/yml.md#variable-substitution
+-   Multiple compose files can be specified, allowing you to override
+    setting in the default compose file. See
+    https://github.com/docker/compose/blob/master/docs/reference/docker-compose.md
+    for more details.
+-   Configuration validation is now a lot more strict
+-   `up` now waits for all services to exit before shutting down
+-   Support for the new docker networking can be enabled with
+    the `--x-networking` flag
+
+New Features
+
+-   Compose can be installed and run as a docker image
+-   `volumes_from` now supports a mode option allowing for read-only
+    `volumes_from`
+-   Volumes that don't start with a path indicator (`.` or `/`) will now be
+    treated as a named volume. Previously this was a warning.
+-   `--pull` flag added to `build`
+-   `--ignore-pull-failures` flag added to `pull`
+-   Support for the `ipc` field added to the compose file
+-   Containers created by `run` can now be named with the `--name` flag
+-   Python 3 support
```

Comment:
"If you install Compose with PyPi or as a Python library, it now works with Python 3."

Most people probably don't know they're using Python. ;)